pkg/api/dashboard.go

```diff
@@ -315,15 +314,16 @@ func dashboardSaveErrorToApiResponse(err error) Response {
 	return Error(500, "Failed to save dashboard", err)
 }
 
-func GetHomeDashboard(c *models.ReqContext) Response {
+// GetHomeDashboard return the home dashboard
+func (hs *HTTPServer) GetHomeDashboard(c *models.ReqContext) Response {
 	prefsQuery := models.GetPreferencesWithDefaultsQuery{User: c.SignedInUser}
-	if err := bus.Dispatch(&prefsQuery); err != nil {
+	if err := hs.Bus.Dispatch(&prefsQuery); err != nil {
```

Comment:
Is it part of some general improvements to use a per-object bus instance instead? I don't know the ramifications here.

Reply:
Yes. Same as with the `settings` package. But it doesn't look like we mention it in the backend readme. 

https://github.com/grafana/grafana/blob/master/pkg/README.md#settings-refactoring